src/Routes/SignIn.jsx

```diff
@@ -74,6 +77,7 @@ const SignIn = ({ isLoading }) => {
           </Button>
         </div>
       </form>
+      <Toaster />
     </div>
   )
 }
```

Comment:
Il y a également une suggestion d'amélioration dans la console concernant le formulaire :

> Non-standard `autocomplete` attribute value
> An autocomplete attribute of a form field uses a non-standard HTML value. This might prevent the browser from correctly autofilling the form.
> To fix this issue, use a valid autocomplete value.

Cet avertissement est dû à l'utilisation d'une valeur non standard pour l'attribut `autocomplete` d'un champ de formulaire. Les valeurs standard pour l'attribut `autocomplete` sont définies par la spécification HTML et sont reconnues par la plupart des navigateurs.

Dans notre cas, on utilise `autocomplete="password"` pour le champ de mot de passe, ce qui n'est pas une valeur standard. La valeur standard pour un champ de mot de passe est `autocomplete="current-password"` pour un champ de mot de passe de connexion, ou `autocomplete="new-password"` pour un champ de mot de passe d'inscription.

Je te laisse corriger cela dans le code pour ce composant et également pour SignUp. 
Tu peux rattacher cette correction à l'issue #31